vendor/bat-native-ledger/src/bat/ledger/internal/static_values.h

```diff
@@ -27,8 +27,11 @@
 #define PUBLISHER_DISTRO_PRODUCTION_SERVER  \
 "https://publishers-distro.basicattentiontoken.org"
 
+#define PREFIX_V1                       "/v1"
 #define PREFIX_V2                       "/v2"
+#define PREFIX_V3                       "/v3"
```

Comment:
Please add comments for usage to end of `#define`

Reply:
I don't see any comment for `PREFIX_V2`. What comment do you expect here?

Reply:
Previously prefixes were only for desktop, however now certain prefixes are android only versions of the API end points

Reply:
NONCRITICAL
Covered in: brave/brave-browser#6235 (25)